en/config/actuators.md

```diff
@@ -470,11 +470,22 @@ To assign an actuator:
 
 ### Motor Configuration
 
+:::note
+If using PWM or OneShot ESCs, you should first perform [ESC Calibration](../advanced_config/esc_calibration.md) (this topic also covers motor configuration).
+
+[DShot](../peripherals/dshot.md) and [DroneCAN](../dronecan/escs.md)/Cyphal ESCs do not require this configuration.
```

Comment:
What do DShot/Dronecan need to do. My understanding is that outputs for disarmed/min/max would not need to be done.

What do we need to check? My assumption is that the motors are spinning in correct direction and the correct motor is mapping to the expected output. But those are covered in a different section of this doc.